app/src/main/java/com/github/siela1915/bootcamp/AutocompleteApi/IngredientAutocomplete.java

```diff
@@ -13,7 +21,8 @@
 
 public class IngredientAutocomplete {
     private final int numberOfIngredients = 5;
-    private final String apiKey = "44a82829c64d4202a18b887e47e76bdd";
+    private final int numberOfNutrition = 3;
+    private final String apiKey = BuildConfig.API_KEY;
```

Comment:
```suggestion
    private final String API_KEY = BuildConfig.API_KEY;
```
Capital letters can be a better naming convention for constant variable.